packages/angular_devkit/core/src/virtual-fs/host/buffer.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { UnsupportedPlatformException } from '../../exception/exception';
+import { FileBuffer } from './interface';
+
+export function stringToFileBuffer(str: string): FileBuffer {
+  // If we're in Node...
```

Comment:
This function only works for node and is only being used in specs. It's still part of `virtual-fs` so it would not be surprising if someone in the future imports it into non-spec code, making it node only.

A comment here would help prevent that, or maybe renaming the file to make it clear it's a spec helper.

Reply:
It's not just a spec helper. It will be used for example to create a `FileBuffer` from a string in schematics (since Tree methods can take both). I'm going to add the non-node version of this.

Reply:
BTW, the StackBlitz folks started using Schematics in the browser, so we already have a case for non-Node implementations. I'm going to start making it harder for us to add Node APIs outside of a `/node` sub-package.